src/extractor/extraction_containers.cpp

```diff
@@ -722,24 +722,25 @@ void ExtractionContainers::PrepareRestrictions()
         if (way_start_and_end_iterator->way_id >
             OSMWayID{static_cast<std::uint32_t>(restrictions_iterator->restriction.from.way)})
         {
-            util::SimpleLogger().Write(LogLevel::logWARNING)
+            util::SimpleLogger().Write(LogLevel::logDEBUG)
```

Comment:
This is fine: Restrictions usually are not that important (other than turn restrictions.)